builds/fabric8-che/plugins/keycloak-plugin-server/pom.xml

```diff
@@ -42,6 +42,11 @@
             <artifactId>openshift-client</artifactId>
         </dependency>
         <dependency>
+            <groupId>io.jsonwebtoken</groupId>
```

Comment:
This is a new dependency. So for now I build this with `-Dskip-enforce=true`. But it should be added in the `che-dependencies` repo and create / submit the corresponding CQs.